src/deployment/deployment-hapi-plugin.ts

```diff
@@ -146,20 +144,19 @@ class DeploymentHapiPlugin {
     }, {
       method: 'GET',
       path: '/raw-deployment-handler/{param*}',
-      handler: this.directoryHandler('key'),
+      handler: this.directoryHandler(PREKEY),
```

Comment:
In some cases the `PREKEY` is passed in as a parameter and in other cases it's not. Should probably be consistent.